Userscripts/faction_last_active.user.js

```diff
@@ -61,37 +64,51 @@ const toggleLastAction = (iconsTitle, memberUL) => {
     if (iconsTitle.innerText === 'Icons') {
         iconsTitle.childNodes[0].nodeValue = 'Last Action'
         get_api().then(res => {
-                if (res.error && res.error.code === 2) alert(`Invalid API key found in ${GM.info.script.name} script. Please update it on line 14.`)
-                for (const li of memberUL.children) {
-                    const lastActionDIV = li.querySelector('.last-action')
+            if (res.error && res.error.code === 2) alert(`Invalid API key found in ${GM.info.script.name} script. Please update it on line 14.`)
```

Comment:
We already desynced the instructions, but it's now on line 16 where the hardcoded instructions is.